starter-code/css/style.css

```diff
@@ -0,0 +1,44 @@
+nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.textImage {
```

Comment:
Don't use position absolute please :) Delete all your `.textImage` CSS, you don't need it anymore to center things with bootstrap :)